tests/conftest.py

```diff
@@ -1,9 +1,31 @@
 # Copyright (C) 2016  Custodia Project Contributors - see LICENSE file
 import warnings
 
+import pytest
+
 from custodia.log import ProvisionalWarning
 
 # silence our own warnings about provisional APIs
 warnings.simplefilter('ignore', category=ProvisionalWarning)
 # deprecated APIs raise an exception
 warnings.simplefilter('error', category=DeprecationWarning)
+
+
+SKIP_INTEGRATION = "--skip-servertests"
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        SKIP_INTEGRATION,
+        action="store_true",
+        help="Skip integration tests"
+    )
+
+
+def pytest_runtest_setup(item):
+    skip = item.config.getoption(SKIP_INTEGRATION)
```

Comment:
It is not marked as skip. This is 'is the skip option enabled on the command line'.